dataStores/router.js

```diff
@@ -0,0 +1,12 @@
+const ChatController = require('./chat/ChatController');
+const chatController = new ChatController();
+const router = require('express').Router();
```

Comment:
Ja bym to dał linijkę wyżej, żeby separować jakoś tworzenie obiektów od importów. Ta linijka byłaby przejściowa:P